serverless.yml

```diff
@@ -137,4 +137,40 @@ functions:
           caching:
             enabled: true
             cacheKeyParameters:
-              - name: request.querystring.id
\ No newline at end of file
+              - name: request.querystring.id
+
+  node_api_get_address_balance:
+    handler: handlers/node_api.get_address_balance
+    package:
+      patterns:
+        - 'handlers/node_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: node_api/thin_wallet/address_balance
```

Comment:
Can't we use only `node_api/address_balance`?

Reply:
We can, but when more apis are migrated to the explorer-service the `node-api` prefix will make it easier to identify which ones are a "proxy" to the full-node

Reply:
Yeah, I agree with `node_api`, I was just thinking about removing `thin_wallet`

Reply:
The thin_wallet is because i wanted to maintain the path structure from the full-node, to make it easier to proxy calls

Reply:
Changed to `node_api/address_balance`